scripts/get_qemu.sh

```diff
@@ -1,30 +1,35 @@
 #!/usr/bin/env bash
+# Download and make qemu-user-static [1] executable in QEMU_DIR
+# for various architectures (archs)
+# [1]: https://github.com/multiarch/qemu-user-static/
+set -e
 
 CURRENT_DIR=$(pwd)
-QEMU_STATIC_VERSION=v3.1.0-3
 QEMU_DIR="./build/qemu/"
+# qemu-user-static version and download URL
+QEMU_STATIC_VERSION=v5.1.0-7
```

Comment:
3->5, is there any danger in this?

Reply:
Frankly I don't know at all. 
However, from my understanding, since it's an emulation layer it's not packaged in the released binaries. It's just a facility to be able to build and run images with different architecture than the host. So I would answer no ... 
But maybe someone with a better knowledge will give a a better explanation.

Reply:
In fact by by digging a little more I realized that the download performed by `get_qemu.sh` is not necessary if we use `qemu-user-static` image instead of `qemu-user-static:register` in **persistent** mode. I've performed the modification. I will update the description of the PR.